<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateSettingsDialog</class>
 <widget class="QDialog" name="UpdateSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Update Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QGroupBox" name="autoUpdateGroup">
     <property name="title">
      <string>Automatic Updates</string>
     </property>
     <layout class="QVBoxLayout" name="autoUpdateLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QCheckBox" name="autoCheckBox">
        <property name="text">
         <string>Automatically check for updates</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="intervalLayout">
        <item>
         <widget class="QLabel" name="intervalLabel">
          <property name="text">
           <string>Check every:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="intervalSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="suffix">
           <string> days</string>
          </property>
          <property name="value">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="intervalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="channelLayout">
        <item>
         <widget class="QLabel" name="channelLabel">
          <property name="text">
           <string>Update channel:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="channelComboBox">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="channelSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="notificationGroup">
     <property name="title">
      <string>Notification Settings</string>
     </property>
     <layout class="QVBoxLayout" name="notificationLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QCheckBox" name="notifyOnlyCheckBox">
        <property name="text">
         <string>Only notify, don't download automatically</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="includeBetaCheckBox">
        <property name="text">
         <string>Include beta versions</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Reset to Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>